src/main/java/christmas/controller/EventController.java

```diff
@@ -0,0 +1,38 @@
+package christmas.controller;
+
+import christmas.model.Event;
+import christmas.model.Order;
+import christmas.util.PrintPhrase;
+import christmas.view.InputView;
+import christmas.view.OutputView;
+
+public class EventController {
+	private OrderController orderController;
+	private InputView inputView;
+	private OutputView outputView;
+
+	public EventController() {
+		orderController = new OrderController();
+	}
```

Comment:
```suggestion
	private final OrderController orderController;
	private final InputView inputView;
	private final OutputView outputView;

	public EventController(OrderController orderController, InputView inputView, OutputView outputView) {
		this.orderController = new OrderController();
		this.inputView = inputView;
		this.outputView = outputView;
	}
```
인스턴스 변수들을 지연 초기화하시면(객체 생성 시 할당 안하고 나중에 할당) 나중에 사용할 때 NPE 발생 위험이 있습니다. final로 선언하고 생성자에서 할당해주시거나 반드시 호출 전에 할당을 보장할 수 있도록 작성해주세요.

근데 아래 코드 보면 inputView, outputView 할당하는 코드가 없는데 호출 시에 NPE 발생하지 않던가요?

Reply:
저는 inputView outputView의 메서드들을 static으로 선언해서 인스턴스 업싱도 메서드 호출할 수 있게 사용했습니다! 어떠한 상태를 가지는 객체들이 아니고 함수처럼 쓸 클래스라 그렇게 해보았습니다! 그런데 다른분들은 다들 input,ouputview객체를 생성하시더라구요